topology/topology.go

```diff
@@ -110,14 +110,14 @@ func (topology *TopologySvc) handleHost(input interface{}, ctx common.RestContex
 	if err != nil {
 		return nil, err
 	}
-	agentUrl := fmt.Sprintf("http://%s:%d", host.Ip, host.AgentPort)
-	agentLink := common.LinkResponse{Href: agentUrl, Rel: "agent"}
+	agentURL := fmt.Sprintf("http://%s:%d", host.Ip, host.AgentPort)
+	agentLink := common.LinkResponse{Href: agentURL, Rel: "agent"}
 	hostLink := common.LinkResponse{Href: hostListPath + "/" + idStr, Rel: "self"}
 	collectionLink := common.LinkResponse{Href: hostListPath, Rel: "self"}
 
 	links := []common.LinkResponse{agentLink, hostLink, collectionLink}
-	hostIdStr := strconv.FormatUint(host.Id, 10)
-	hostMessage := common.HostMessage{Id: hostIdStr, RomanaIp: host.RomanaIp, Ip: host.Ip, Name: host.Name, AgentPort: int(host.AgentPort), Links: links}
+	hostIDStr := strconv.FormatUint(host.Id, 10)
```

Comment:
Sad how Go's coding/naming standards force you to make identifiers less readable. Sigh...